code/ui/Scoreboard.cs

```diff
@@ -175,6 +173,26 @@ private ScoreboardGroup GetScoreboardGroup(PlayerScore.Entry entry)
                 // TODO better spectator check, maybe with a player var
                 group = "Dead";
             }
+            else
+            {
+                ulong steamId = entry.Get<ulong>("steamid", 0);
+
+                if (steamId != 0)
+                {
+                    foreach (Client client in Client.All)
+                    {
+                        if (client.Pawn.IsValid() && client.SteamId == steamId)
+                        {
+                            if ((client.Pawn as TTTPlayer).IsMissingInAction)
+                            {
+                                group = "MIA";
```

Comment:
We might want to make these static strings at the top of the file, or, if we decide with the enum change I proposed, you can just toString() the enum value.